Tests/test_lib_pack.py

```diff
@@ -265,9 +266,11 @@ def test_LA(self):
     def test_P(self):
         self.assert_unpack("P", "P;1", b'\xe4', 1, 1, 1, 0, 0, 1, 0, 0)
         self.assert_unpack("P", "P;2", b'\xe4', 3, 2, 1, 0)
-        # self.assert_unpack("P", "P;2L", b'\xe4', 1, 1, 1, 0)  # erroneous?
```

Comment:
This line was not too long, I have just changed it for consistency with the next change